### PR TITLE
Co-author: Drizzle

### PR DESCRIPTION
**Pull request by haydenbleasel**
This pull request includes updates to the documentation for switching databases in the `next-forge` project. The primary focus is on adding a guide for switching from Prisma to Drizzle and renaming the existing database guide to focus on PlanetScale.

### CODE REVIEW

File: apps/_docs/content/docs/drizzle.mdx [hunk 1]
```diff
@@ -0,0 +1,112 @@
+---
+title: Drizzle
+description: How to switch from Prisma to Drizzle.
+---
+
+next-forge uses [Prisma](https://prisma.io) as its ORM by default. However, you can easily switch to another ORM like [Drizzle](https://drizzle.dev) or [Kysely](https://kysely.dev/).
```
Comment (AlexBlokh) [inline apps/_docs/content/docs/drizzle.mdx:6]: orm.drizzle.team 🫡

File: apps/_docs/content/docs/drizzle.mdx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+config({ path: '.env' });
+
+const databaseUrl = process.env.DATABASE_URL;
+
+if (!databaseUrl) {
+  throw new Error('Missing DATABASE_URL environment variable.');
+}
+
+const client = neon(databaseUrl);
```
Comment (AlexBlokh) [inline apps/_docs/content/docs/drizzle.mdx:51]: as of our latest API improvements - you can do
```ts
import { drizzle } from "drizzle-orm/neon-http";

const db = drizzle(process.env.DATABASE_URL);
```

File: apps/_docs/content/docs/drizzle.mdx (hunk 1 above)
Comment (haydenbleasel) [reply apps/_docs/content/docs/drizzle.mdx:6]: Lol Cursor blindsiding me

File: apps/_docs/content/docs/drizzle.mdx (hunk 2 above)
Comment (haydenbleasel) [reply apps/_docs/content/docs/drizzle.mdx:51]: Does this need to be updated? https://orm.drizzle.team/docs/tutorials/drizzle-with-neon#connect-drizzle-orm-to-your-database

**Pull request merged**